resources/i18n/en.json

```diff
@@ -1194,6 +1194,7 @@
     "Name": "Name",
     "AllowedHosts": "Allowed hosts",
     "CannotCreateResourcePolicy": "Cannot create Resource Policy. Please check input values.",
+    "CannotUpdateResourcePolicy": "Cannot update Resource Policy. Please check input values.",
```

Comment:
We couldn't say "Please check input values" because this PR might display this message even when the user input is correct.